src/components/data-table.jsx

```diff
@@ -33,15 +33,17 @@ const DataTable = (props) => {
             {(cell_value, index) => (
               <tr
                 draggable="true"
-                onTouchStart={() => setActiveIndex(index())}
+                onTouchStart={() =>
+                  setActiveIndex(`${index()}-${cell_value.tick}`)
```

Comment:
```suggestion
                  setActiveIndex(index() - cell_value.tick)
```

Reply:
Since index() is a num and cell_value.tick is a string, the resultant will be NaN, which then wont work

Reply:
Nit: I think you can  use only cell_value.tick, no need to use combination of the index & the symbol.

Reply:
updated